package.json

```diff
@@ -175,10 +176,11 @@
 		"jest-transform-graphql": "^2.1.0",
 		"memory-fs": "^0.5.0",
 		"ora": "^4.0.2",
-		"postcss": "^7.0.35",
-		"postcss-loader": "^3.0.0",
+		"postcss": "^8.3.0",
+		"postcss-loader": "^4.3.0",
 		"rimraf": "^3.0.2",
-		"sass-loader": "^8.0.2",
+		"sass": "^1.32.13",
```

Comment:
There's a more recent version of this, but the more recent version has deprecated using `/` for division, in favor of `math.div()`, and it generates several hundred deprecation warnings in the build log. I tried using the options to suppress those warnings, but the options did not seem to work.